docs/tutorials/services/webcam-line-follower-robot.md

```diff
@@ -66,28 +66,31 @@ Follow the instructions until the Viam app shows that your robot has successfull
 {{% tab name="Builder UI" %}}
 
 Navigate to the **Config** tab of your robot's page in [the Viam app](https://app.viam.com).
-Click on the **Components** subtab and navigate to the **Create component** menu.
+Click on the **Components** subtab.
 
-1. **Add the board.**
+1. **Add the [board](/components/board/).**
```

Comment:
nit: same nit, let's move hyperlinks to body copy.

Reply:
```suggestion
1. **Add the board.**
```